addons-l10n/en/message-filters.json

```diff
@@ -0,0 +1,11 @@
+{
+  "message-filters/messages": "I would like to see messages from:",
```

Comment:
this sounds odd. "I would like to see messages from studio activity"
i think it should be "I would like to see studio activity messages"

Reply:
I think it's fine in the context of the other elements. Plus, we can't just insert "studio activity" into the sentence.